pkg/apis/contrail/v1alpha1/tests/config_test.go

```diff
@@ -476,7 +492,10 @@ func SetupEnv() Environment {
 		Items: vrouterPodItems,
 	}
 
+	configApiUrl := "https://" + configService.Spec.ClusterIP + ":" + strconv.Itoa(v1alpha1.ConfigApiPort)
+	telemetryUrl := "http://" + configService.Spec.ClusterIP + ":" + strconv.Itoa(v1alpha1.AnalyticsApiPort)
```

Comment:
Why is it `http` not `https`?

Reply:
I took this as my reference https://github.com/Juniper/contrail-operator/blob/2b8f86038699426b893f5c3e0f02dfdd6e0c5af9/pkg/controller/command/command_config_maps.go#L37

Reply:
Changed to https